contents/blog/product-led-growth.md

```diff
@@ -0,0 +1,193 @@
+
+### Communication
+Listen to your users on everything.
+
+Built something? Get feedback. New idea? Get feedback. Launched a release? Get feedback.
+
+As one of our core devs put it following interviews with users about a new feature he built: "Your assumptions are mostly wrong. Talk to people to correct them".
+
+When generating growth primarily through Sales or Marketing, the segregation between the people who are building the product and those selling it can lead to diversion between what users want and what is being offered to them.
+
+When focusing on the product, you should always be touching base with your users, and be willing to drop features you love if they don't feel the same way.
+
+### Priorities
+
+At PostHog, our methodology is:
+
+<blockquote class="blog-quote">
+
+If we keep our team first and users second, then our investors will take care of themselves!
+
+</blockquote>
+
+Users are key and should be a top priority.
+
+However, you must remember that your team consists of the first users of your product, its main advocates, and its builders! 
+
+Hence, since users are a priority, and the team takes care of the users, you need to take care of your team!
+
+### Growth
+Since you're not aggressively seeking out users with Product-Led Growth, you are likely to experience slower growth at first. 
+Later on, however, if you did truly build something great, you may benefit from a network effect that skyrockets your userbase. 
+
+Facebook and Slack are great examples of this. 
+
+### Revenue 
+Since the initial focus is on building a product rather than selling it, you will likely make less money in the short-run.
+
+With the goal not being to hit a Sales metric, you might be turning down opportunities to generate revenue if they are not perfectly aligned with what you're building.
+
+As such, you need to make sure your team and investors are aligned with this. It should be understood that it's okay to not be making money in the early stages.
+
+## One Size Doesn't Fit All
+Product-Led Growth is not a magic pill. It does have its shortcomings.
+
+Primarily, how can you let your product speak for itself if you can't get anyone to look at it in the first place?
+
+But rather than tell you when Product-led Growth might not work, I'll tell you what aspects can be beneficial to it.
+
+### Funding
+It's much easier to focus almost exclusively on building a product when you're already well-funded. This is our case at PostHog.
+
+To build a great product, you need a great team. And to build a great team, you need money. 
+
+Thus, if you're a company without a lot of money in the bank, you might just need to get some Sales done now, simply to fund your operations.
+
+Not having funding shouldn't discourage you from trying a PLG approach, however. It can still work, just look at GitLab. They first raised money after already having 100,000 users and Sid (their CEO), claims they didn't even have a significant marketing budget until then. Wow. 
```

Comment:
"claims" makes it sound like we don't believe him, which we do :) Also, link to interview